packages/appeals-service-api/__tests__/unit/services/appeals-case-data.service.test.js

```diff
@@ -49,7 +50,15 @@ describe('src/services/lpa-dashboard-appeals.service.test', () => {
 
 		const result = await getAppeals(lpaCode);
 
-		expect(collectionMock.find).toHaveBeenCalledWith({ LPACode: lpaCode }, appealsProjection);
+		expect(collectionMock.find).toHaveBeenCalledWith(
+			{
+				LPACode: lpaCode,
+				validity: 'Valid',
+				questionnaireDueDate: { $regex: regEx },
+				questionnaireReceived: { $not: regEx }
+			},
```

Comment:
initially tried an 'and' for due date - **is not empty or null && matches regex**, and an 'or' for received **is empty or null || does not match regex** but I don't think mongo does short circuit evaluation, so this seems pointless.

Reply:
have replaced with checking for date type